api/openapi.yaml

```diff
@@ -319,6 +319,10 @@ components:
   schemas:
     Test:
       type: object
+      required:
```

Comment:
Why are you using required here? I tried to do this in the backend, and the validation is stupidly strict, like a bool `false` is parsed as `empty` and fails the validation in the backend. I would leave this as it was, unless there is a blocking need for it